var/ramble/repos/builtin/applications/hmmer/application.py

```diff
@@ -34,15 +34,16 @@ class Hmmer(SpackApplication):
 
     software_spec('hmmer', spack_spec='hmmer@3.3.2', compiler='gcc9')
 
+    # This would ideally not use the current_relase, as the package will need to be manually updated per release
```

Comment:
If possible, can you document what the current version of "current_release" is, so we don't have to figure this out in the future?

Reply:
Done (its Pfam36.0)